### ISSUE TITLE
Validate YAML files

### ISSUE THREAD

**Issue by ruflin** [elastic/package-registry#422]
All available yaml files in the packages should be validated for valid yaml to make sure packages only contain valid yaml.

An example where this would have been useful can be found here: https://github.com/elastic/package-registry/pull/421

**Comment by ruflin** [comment]
If we validate all yaml files, we should also validate all json files.

**Comment by ph** [comment]
@ycombinator I think its part of the validation that you are currently enforcing?

**Comment by ycombinator** [comment]
Yes, this will ultimately be handled by the validation provided by the package-spec.

**Comment by ph** [comment]
Going to move the issue in that repository.
